node/api.md

```diff
@@ -325,21 +325,13 @@ struct NewWalletRequest {
 }
 ```
 
-Response:
-
-```rust
-struct NewWalletResponse {
-    id: [u8; 32],
-}
-```
-
-### /wallet/:id/balance
+### /wallet/balance
 
 Returns wallet's balance.
 
 Request:
 
-`GET /wallet/:id/balance`
+`GET /wallet/balance`
```

Comment:
Same change needed in the table of contents (in the top of this file).

Reply:
Fixed